core/controllers/resources.py

```diff
@@ -137,18 +190,20 @@ class PromoBarHandler(base.BaseHandler):
     REDIRECT_UNFINISHED_SIGNUPS = False
 
     @acl_decorators.open_access
-    def get(self):
+    def get(self) -> None:
         self.render_json({
             'promo_bar_enabled': config_domain.PROMO_BAR_ENABLED.value,
             'promo_bar_message': config_domain.PROMO_BAR_MESSAGE.value
         })
 
     @acl_decorators.can_access_release_coordinator_page
-    def put(self):
-        promo_bar_enabled_value = self.normalized_payload.get(
-            'promo_bar_enabled')
-        promo_bar_message_value = self.normalized_payload.get(
-            'promo_bar_message')
+    def put(self) -> None:
+        assert self.user_id is not None
+        assert self.normalized_payload is not None
+        promo_bar_enabled_value = self.normalized_payload[
+            'promo_bar_enabled']
+        promo_bar_message_value = self.normalized_payload[
+            'promo_bar_message']
```

Comment:
```suggestion
        promo_bar_enabled_value = self.normalized_payload['promo_bar_enabled']
        promo_bar_message_value = self.normalized_payload['promo_bar_message']
```

I think these can be in a single line since `get` is taken away

Reply:
Done!